doc/source/serve/deployment-graph/model_composition.md

```diff
@@ -0,0 +1,273 @@
+(serve-model-composition)=
+
+# Model Composition: Deployment Graphs
+
+```{note}
+Note: This feature is in Alpha, so APIs are subject to change.
+```
+
+This section helps you:
+
+* connect your Ray Serve deployments together with the **deployment graph** API
+* serve your applications that use multi-model inference, ensemble models, ML model composition, or mixed business logic/model inference workloads
+* independently scale and configure each of your ML models and business logic steps
+
+Ray Serve's **deployment graph** API lets you compose your deployments together by describing how to route a request through your deployments. This is particularly useful if you're using ML model composition or mixing business logic and model inference in your application. You can encapsulate each of your models and each of your business logic steps in independent deployments. Then, you can connect these deployments together in a deployment graph.
+
+## Implicit Composition: Cross-Deployment Calls
```

Comment:
I don't think users will understand what "implicit" vs "explicit" means in this context. let's just "call it what it is" instead:
```suggestion
## Calling Deployments using ServeHandle
```

Reply:
this more closely matches my changes to user guide as well